### PR TITLE
feat: support follow/show logs on Podman

### LINKED ISSUE #549: Support follow/show logs on podman
[kind/Feature]

## Which functionality do you think we should add?
support show log and follow logs when running applications on podman

## Why is this needed?
supported in odo 3.7.0 (see https://odo.dev/blog/odo-v3.7.0#odo-logs-on-podman)

### PR DESCRIPTION
**Pull request by sbouchet**
## What is the purpose of this change? What does it change?
with this change, when component is running on podman, show/follow logs are correctly chowing logs from podman

## Was the change discussed in an issue?
fixes #549

### CODE REVIEW

**Review by adietish — APPROVED** [review]
worked for me deploying to podman desktop. I'd give a serious thought about simplifying or unifying the very complex boolean expressions in `ShowLogComponentAction`

File: src/main/java/org/jboss/tools/intellij/openshift/actions/component/ShowLogComponentAction.java [hunk 1]
```diff
@@ -117,4 +117,11 @@ private Optional<Boolean> isDeploy(Odo odo, ComponentNode componentNode) throws
     }
     return result;
   }
+
+  private String getPlatform(Component component){
+    if (component.getLiveFeatures().is(ComponentFeature.DEV_ON_PODMAN)){
+      return ComponentFeature.Constants.PODMAN;
+    }
+    return null;
```
[the remark this answers is not in the document]
Comment (sbouchet) [reply src/main/java/org/jboss/tools/intellij/openshift/actions/component/ShowLogComponentAction.java:125]: no, it's when component is on dev mode on cluster. for now, odo support --platform flag for podman only, and no falg for any remote cluster defined in kubeconfig

**Comment by sbouchet** [comment]
@adietish refactored most of your comments, can you re-review please ?

**Review by adietish — APPROVED** [review]
LGTM. Works well for me. 
I was a bit astonished though when "Follow Log" and "Show Log" were not present in the context menu any more. Took me a bit to grasp that those are missing when there's a terminal (process) with the logs. I tend to think that it would have been better to have them present and bringing the existing terminal to the front in that case.

**Pull request merged**